src/pybind/mgr/cephadm/tests/test_cephadm.py

```diff
@@ -1563,7 +1563,7 @@ def test_apply_prometheus(self, spec: PrometheusSpec, raise_exception: bool, msg
             if not raise_exception:
                 cephadm_module._apply(spec)
             else:
-                with pytest.raises(OrchestratorError, match=msg):
+                with pytest.raises(SpecValidationError, match=msg):
```

Comment:
given that the exception now happens as part of the service spec code rather than actually in `_apply`, perhaps this test ought to be moved to `ceph/src/python-common/ceph/tests/test_services`?